projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -848,6 +870,78 @@ public function render_telephone_field( $id, $label, $value, $class, $required,
 		return $field;
 	}
 
+	/**
+	 * Return the HTML for the telephone field.
+	 *
+	 * @param int    $id - the ID.
+	 * @param string $label - the label.
+	 * @param string $value - the value of the field.
+	 * @param string $class - the field class.
+	 * @param bool   $required - if the field is marked as required.
+	 * @param string $required_field_text - the text in the required text field.
+	 * @param string $placeholder - the field placeholder content.
+	 *
+	 * @return string HTML
+	 */
+	public function render_phone_field( $id, $label, $value, $class, $required, $required_field_text, $placeholder ) {
+		$this->set_invalid_message( 'phone', __( 'Please enter a valid phone number', 'jetpack-forms' ) );
+		$label = $this->render_label( 'phone', $id, $label, $required, $required_field_text );
+
+		// since now the telephone field can include a country selector, we need to handle the rendering of the field differently
+		$styles = '';
+
+		if ( ! empty( $this->field_styles ) ) {
+			$styles .= sprintf( 'style="%s" ', esc_attr( $this->field_styles ) );
+		}
+
+		if ( ! is_string( $value ) ) {
+			$value = '';
+		}
+
+		$id               = esc_attr( $id );
```

Comment:
instead of esc_attr is here. lets esc_attr it late. 
This way we don't pass the esc_value into functions and double esc_attr(). Such as on line 941.